index.js

```diff
@@ -49,8 +49,8 @@ ansiEscapes.cursorForward = (count = 1) => ESC + count + 'C';
 ansiEscapes.cursorBackward = (count = 1) => ESC + count + 'D';
 
 ansiEscapes.cursorLeft = ESC + 'G';
-ansiEscapes.cursorSavePosition = ESC + (isTerminalApp ? '7' : 's');
-ansiEscapes.cursorRestorePosition = ESC + (isTerminalApp ? '8' : 'u');
+ansiEscapes.cursorSavePosition = ESC + (isTerminalApp ? '\u001B7' : 's');
+ansiEscapes.cursorRestorePosition = ESC + (isTerminalApp ? '\u001B8' : 'u');
```

Comment:
I think you want to move `ESC` to the "else" here.

Reply:
Ah yes, good catch. I've updated the PR. And thanks for the quick reply!